tribe-ext-map-show-all-events-with-same-venue/src/tribe-events-ajax-maps.js

```diff
@@ -0,0 +1,711 @@
+/**
+ * @file This file contains all map view specific javascript.
+ * This file should load after all vendors and core events javascript.
+ * @version 3.0
+ */
+
+(function( window, document, $, td, te, tf, tg, ts, tt, config, dbug ) {
```

Comment:
Throughout this file the tabulation is off. Just as in the original, we should use tabs for the initial indent and spaces for any alignment after that.